ogr/services/github/comments.py

```diff
@@ -0,0 +1,51 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from github import IssueComment as _GithubIssueComment
+
+from ogr.abstract import IssueComment, PRComment
+
+
+class GithubIssueComment(IssueComment):
+    def __init__(self, raw_comment: _GithubIssueComment) -> None:
```

Comment:
I would save this object as a "private" property. What do you think?

(So we can update the comments in the future. -- e.g., I would like to have some method for hiding comments)

Reply:
You mean not exporting it and keeping only the Issue/PR Comment exposed?

Reply:
I mean `raw_comment` will be useful.
```suggestion
    def __init__(self, raw_comment: _GithubIssueComment) -> None:
        self.__raw_comment = raw_comment
```

Reply:
So we can implement some methods using this `__raw_comment`.

Reply:
ok